src/Components/Header/Header.js

```diff
@@ -57,4 +81,15 @@ const Header = () => {
   )
 }
 
-export default Header
\ No newline at end of file
+const mapStateToProps = () => {
+    return {
+        userStatus: localStorage.getItem('token'),
```

Comment:
please what is this piece of code?? i don't understand

Reply:
please get the user login status from the user object on store not if a token is present. what if the token is expired. and why is getting the token event in the mapStateToProps??